functionaltests/apm_logs_filters.go

```diff
@@ -28,24 +28,26 @@ var (
 			"message": {Query: "http: TLS handshake error from 127.0.0.1:"},
 		},
 	}
+
 	esReturnedUnknown503 = types.Query{
 		MatchPhrase: map[string]types.MatchPhraseQuery{
 			"message": {Query: "ES returned unknown status code: 503 Service Unavailable"},
 		},
 	}
-	refreshCache503 = types.Query{
+
+	refreshCache403 = types.Query{
 		MatchPhrase: map[string]types.MatchPhraseQuery{
-			"message": {Query: "refresh cache elasticsearch returned status 503"},
+			"message": {Query: "refresh cache elasticsearch returned status 403"},
```

Comment:
I don't think we should ignore the `403` as it indicates a permission issue, which in unexpected and should be investigated.